app/uk/gov/hmrc/agentsubscriptionfrontend/controllers/SubscriptionController.scala

```diff
@@ -305,18 +263,6 @@ class SubscriptionController @Inject()(
 
   private def completeMappingWhenAvailable(utr: Utr, completedPartialSub: Boolean = false)(
     implicit request: Request[AnyContent],
-    hc: HeaderCarrier): Future[Result] = {
-
-    val doMappingAnswer: Boolean = request.session.get("performAutoMapping").contains("true") || completedPartialSub
-
-    for {
-      _ <- {
-        if (appConfig.autoMapAgentEnrolments && doMappingAnswer)
-          mappingConnector.updatePreSubscriptionWithArn(utr)
-        else {
-          toFuture(())
-        }
-      }
-    } yield Redirect(routes.SubscriptionController.showSubscriptionComplete())
-  }
+    hc: HeaderCarrier): Future[Result] =
+    Redirect(routes.SubscriptionController.showSubscriptionComplete())
```

Comment:
just directly redirect to subscription complete instead of calling this and get rid of this method